.github/workflows/validate.yml

```diff
@@ -14,17 +13,29 @@ jobs:
         run: |
           git fetch origin main:localmain
 
-          # Gets changed json files excluding the json schema, right now only validating definition.json
-          export FILES=$(git diff --name-only localmain | grep definition.json)
+          # Gets changed definition files
+          # TODO: validate all config files and not just trial's definition.yml
+          export FILES=$(git diff --name-only localmain | grep definition.yml)
 
           echo "Files changed since last commit..."
           echo "$FILES"
 
           echo "::set-output name=diff::$( echo "$FILES" | tr '\n' ',' | sed '$s/,$/\n/' )"
 
       - name: Validate JSON
-        if: ${{ steps.calc_files_diff.outputs.diff != '' }}
+        # if: ${{ steps.calc_files_diff.outputs.diff != '' }}
+        if: always()
         uses: docker://orrosenblatt/validate-json-action:latest
         env:
           INPUT_SCHEMA: .ci/trial-json-schema.json
-          INPUT_JSONS: ${{ steps.calc_files_diff.outputs.diff }}
+          # INPUT_JSONS: ${{ steps.calc_files_diff.outputs.diff }}
+          INPUT_JSONS: sample_trial/definition.json
+      
+      - name: Validate YML
+        if: always()
+        uses: nrkno/yaml-schema-validator-github-action@master
+        with:
+          schema: '.ci/trial-json-schema.json'
+          target: 'sample_trial/definition.yml'
+          # Uncomment to enable strict checks
```

Comment:
if we can lets enable. otherwise remove these two lines

Reply:
Resolved, earlier version